media/js/products/vpn/experiment-refresh-pricing-layout.es6.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import TrafficCop from '@mozmeao/trafficcop';
+import { isApprovedToRun } from '../../base/experiment-utils.es6.js';
+
+const href = window.location.href;
+
+const initTrafficCop = () => {
+    if (
+        href.indexOf(
+            'entrypoint_experiment=vpn-refresh-pricing&entrypoint_variation='
+        ) !== -1
+    ) {
+        if (href.indexOf('entrypoint_variation=1') !== -1) {
+            window.dataLayer.push({
+                'data-ex-variant': 'vpn-refresh-pricing-columns',
+                'data-ex-name': 'vpn-refresh-pricing'
+            });
```

Comment:
There's a slight issue here in that these dataLayer push events are never triggered. It took me a hot minute to figure out why, but I think it's because we include both `entrypoint_variation` and `entrypoint_experiment` in the [list of excluded params](https://github.com/mozilla/bedrock/blob/main/media/js/base/experiment-utils.es6.js#L7) in `isApprovedToRun()`.

We did this originally to avoid clobbering upstream experiments that might be run by other teams I think, but my memory is fuzzy. Perhaps we should revisit if that caution still makes sense? I'm not too sure tbh.

Still, I think you can likely fix this easily enough by moving these events outside of the `initTrafficCop()` function.

Reply:
Hmm the previous landing page experiment was set up the same way, so I guess those events never fired? Are we even sure they're necessary? I wonder if we can just remove them if nobody is looking at them anyway.